README.md

```diff
@@ -100,6 +100,8 @@ if let j: AnyObject = json {
 
 For more information, see the [Documentation](Documentation/)
 
+Please note: the above example requires an additional, external module named [Curry](https://github.com/thoughtbot/Curry) to automatically curry `User.init`.
```

Comment:
also it's not automatic so we should say "to use the `curry` function to curry `User.init`." or something like that